code/game/objects/machinery/vending/new_marine_vendors.dm

```diff
@@ -479,9 +479,9 @@
 /obj/machinery/marine_selector/clothes/synth
 	name = "M57 Synthetic Equipment Vendor"
 	desc = "An automated synthetic equipment vendor hooked up to a modest storage unit."
-	icon_state = "synth"
-	icon_vend = "synth-vend"
-	icon_deny = "synth-deny"
+	icon_state = "snowflake"
```

Comment:
why funny name

Reply:
Req's opertional vendor has synth. We would have to change req's eperational vendor icon path if we want synth vendor to have name.